setup.py

```diff
@@ -93,6 +97,7 @@ def run_tests(self):
 tests_require = ["pytest", "pytest-cov", "pytest-timeout>=0.3"]
 install_requires = [
     "pathtools3>=0.2.0",
+    "monotonic>=1.5",
```

Comment:
Could you make the requirement for Python 2 only? And add the `monotonic()` export into `compat.py` instead. Like that, on Python 3 there will not be a useless requirement to install.

Reply:
Sure, I can do that.

Reply:
Actually, `monotonic` is only used in `test_delayed_queue`, so I'll just move it into `tests_require`.